Backstory

hedvigf opened the pull request:
Created backstory, added buttons to show and hide the story and merged it all with the latest version of main scene.

hedvigf committed:
Backstory with buttons

hedvigf committed:
Text fix and resolution improvement

hedvigf committed:
Merge branch 'master' into backstory

hedvigf committed:
Backstory merge with main scene

urbanturban commented:
"Backstory" scene looks awesome, nice work! :) 
Noticed the buttons and backstory sprite didn't show up work/properly in the main scene. I followed the layout in "Backstory" scene and fixed the remaining issues.

urbanturban committed:
Hotfix to Backstory UI merge

- Fixed the overlay issues: The backstory UI and buttons are now infront of everything
- Layout and functionality in mainscene now like Backstory scene by Hedvig

hedvigf commented:
Ah, this was the issue I sat with for several hours. Never tried to put the sprite into the button parent. Great solution!
